client/components/tinymce/plugins/mentions/suggestions-mixin.jsx

```diff
@@ -0,0 +1,206 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Suggestion from './suggestion';
+
+const stopEvent = function( event ) {
+	if ( this.state.suggestionsVisible ) {
+		event.stopPropagation();
+		event.preventDefault();
+	}
+};
+
+const SuggestionsMixin = {
+	componentWillMount: function() {
+		this.suggestions = [];
+		this.suggestionsAbove = false;
+	},
+
+	componentDidMount: function() {
+		this.isMounted = true;
```

Comment:
`isMounted()` exists in core react but is an anti-pattern. Why is it needed here?